dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
Can you tell the marathon team that this is missing in the docs?  Or file a Github issue in the marathon project?

We should be checking the response status here, and forming the appropriate error message.  Catching this exception is a hack.  Also, you're surfacing the JSON parsing error, which is not what the user cares about.